README.md

```diff
@@ -26,15 +30,33 @@ docker-compose -f docker-compose.test.yml up --abort-on-container-exit
 
 The Control Panel app requires Python 3.6+
 
-It is best to use a virtual environment to install dependencies, eg:
+It is best to use a virtual environment to install python dependencies, eg:
 ```sh
 python -m venv venv
 . venv/bin/activate
 pip install -r requirements.txt
 ```
 
+### Kubernetes setup
+
+You need a kubeconfig (~/.kube/config) with the credentials for your k8s cluster. Once you've got that, this should work:
+```sh
+kubectl cluster-info
+```
+
+You need to tell Helm to use the AP charts:
+```sh
+helm repo add mojanalytics https://ministryofjustice.github.io/analytics-platform-helm-charts/charts/
+helm repo update
+```
+
 ### <a name="env"></a>Environment variables
 
+#### Bare minimum (for local testing)
+export DEBUG=True
```

Comment:
You may also want to wrap these commands (and the others added) in a <tt>```sh</tt> block as well.

Not a big deal really so don't worry too much but it may make skimming through this document a bit more pleasant :)